test/e2e_env/multizone/healthcheck/healthcheck.go

```diff
@@ -20,12 +20,17 @@ func ApplicationOnUniversalClientOnK8s() {
 		err := env.Global.Install(MTLSMeshUniversal(meshName))
 		Expect(err).ToNot(HaveOccurred())
 
+		err = DeleteAllResourcesUniversal(*env.Global.GetKumactlOptions(), mesh.RetryResourceTypeDescriptor, meshName)
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
It almost feels like we should have this by default in our meshes. This "safe" defaults are likely hiding bugs in e2e tests don't you think? There's an issue for this already in fact :) https://github.com/kumahq/kuma/issues/3346

Reply:
Yes, I also feel like retry should not be enabled by default at least in E2E tests.

Added a comment to linked issue